### ISSUE TITLE
TravelPlanner Leaderboard Eval Error

### ISSUE THREAD

**Issue by ctarnold** [OSU-NLP-Group/TravelPlanner#44]
Hi,

Thanks for your previous help on the ReAct agent! I'm trying to run eval with a _small_ subset of the test set. Just about 10 examples before I scale up. The [HuggingFace Leaderboard](https://huggingface.co/spaces/osunlp/TravelPlannerLeaderboard) looks most convenient - I'm not trying to make a public submission. I get an error when uploading a jsonl file, as shown in [my forked repo](https://github.com/ctarnold/TravelPlanner/blob/main/evaluation/json/eval.jsonl). I've also tried removing my placeholder for index 10 (I haven't computed it yet). My guess is there's some assertion that length matches the eval set, or perhaps my json is formed incorrectly? 

If there's any assertions on the backend of the eval leaderboard that would help me, please let me know. I'm not trying to submit publicly yet (and not sure if I will, GPU compute is hard to come by), just run preliminary eval. Otherwise, I will try running eval locally using the files you provide and make sure my json lines are formed properly. 

Thank you!

Edit: I've been running one-off queries and saving results to my jsonl file linked, so it looks different now, but the principle is the same. I'm wondering whether the HuggingFace eval automatically blocks file lengths that don't match the given test/train/validation set sizes.

**Comment by ctarnold** [comment]
Hi, I realized on the GitHub docs that offline _test_ eval is not supported. I ended up doing eval of the subset of the test set by padding the file with extra lines to length 1000 of the following form: `{"idx": 100, "query": "Natural Language Query", "plan": [{"day": 1, "current_city": "from [City A] to [City B]", "transportation": "Flight Number: XXX, from A to B", "breakfast": "Name, City", "attraction": "Name, City;Name, City;...;Name, City;", "lunch": "Name, City", "dinner": "Name, City", "accommodation": "Name, City"}, {"day": 2, "current_city": "City B", "transportation": "-", "breakfast": "Name, City", "attraction": "Name, City;Name, City;", "lunch": "Name, City", "dinner": "Name, City", "accommodation": "Name, City"}]}` . This allows eval on a subset, closing the issue.
